bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/energy/Home.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.api.dto.energy;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * @author Bernhard Kreuz - Initial contribution
+ */
+public class Home implements Serializable {
+
+    private String id;
+    private List<Module> modules = new ArrayList<>();
+    private List<Room> rooms = new ArrayList<>();
+    private Map<String, Object> additionalProperties = new HashMap<>();
```

Comment:
I'm curious of what do you collect in additionalProperties. I did not find it in the API.

Reply:
All part of Bernhard's contribution, he used some code-generator to create classes and methods out of the JSON. I did not touch that part.
For me this looks like some sort of failsafe for unexpected stuff coming from the JSON. Not sure if that can be safely removed depending on the deserialization he used (different from how you done it for the other parts of the API).
I'd need some advise here as I am not that savvy in Java.

Reply:
Yes, I understood that browsing the PR and I understand your concerns, but a big part of the work on this PR as been to try to fold Netatmo API answers in a common scheme to avoid duplicated code (and this is quite hard because Netatmo API is far from being very well enginereed).